modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -10195,6 +10291,31 @@ public void testPutCustomObjectEntryWithNestedCustomObjectEntriesInOneToManyRela
 			_OBJECT_FIELD_NAME_2, _NEW_OBJECT_FIELD_VALUE_1);
 	}
 
+	@Test
+	public void testPutScopeScopeKeyPageWithSiteExternalReferenceCode()
```

Comment:
Could you let me know where you got the name of this method?
The method I saw the name is `putScopeScopeKeyByExternalReferenceCode` so the name should be `testPutScopeScopeKeyByExternalReferenceCode`

Can you add the three scenarios (ERC, ID, and name)?